Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM quay.io/centos/centos:stream8 as arcaflow-elasticsearch-plugin
+
+RUN dnf -y module install python39 && dnf -y install python39 python39-pip git
+RUN mkdir /app
+
+COPY LICENSE /app
+COPY es_plugin.py /app
+COPY es_schema.py /app
+COPY requirements.txt /app
+WORKDIR /app
+
```

Comment:
can you add this line `ADD https://raw.githubusercontent.com/arcalot/arcaflow-plugins/main/LICENSE /app`

Reply:
That method of copying the required license is not currently supported by our [manual integration tool](https://github.com/arcalot/arcaflow-plugin-image-builder).

Reply:
Changed.